lib/common/Makefile

```diff
@@ -0,0 +1,12 @@
+CC ?= gcc
```

Comment:
@andrew-elder Is this the file/line you mentioned causing a build problem for you in some context? I don't remember the details of what you said or if it's something that needs fixing before we merge.